tools/protoprint/protoprint.bzl

```diff
@@ -0,0 +1,53 @@
+load("//tools/api_proto_plugin:plugin.bzl", "api_proto_plugin_aspect", "api_proto_plugin_impl")
+
+def _protoprint_impl(target, ctx):
+    return api_proto_plugin_impl(
+        target,
+        ctx,
+        "pproto",
+        "protoprint",
+        [".proto"],
+    )
+
+# Bazel aspect (https://docs.bazel.build/versions/master/starlark/aspects.html)
+# that can be invoked from the CLI to perform API transforms via //tools/protoprint for
+# proto_library targets. Example use:
+#
+#   bazel build //api --aspects tools/protoprint/protoprint.bzl%protoprint_aspect \
+#       --output_groups=proto
+#
+protoprint_aspect = api_proto_plugin_aspect(
+    "//tools/protoprint",
+    _protoprint_impl,
+    use_type_db = True,
+    extra_inputs = ["//.clang-format"],
```

Comment:
not sure if this is necessary - i was struggling to get the clang-format file into the protoprint job